ui/admin/app/templates/scopes/scope/aliases/alias/index.hbs

```diff
@@ -23,7 +23,36 @@
       </nav.link>
     </Rose::Nav::Tabs>
   </page.navigation>
+  {{#let
+    (hash
+      canRead=(can 'read model' @model) canDelete=(can 'delete model' @model)
+    )
+    as |perms|
+  }}
+    <page.actions>
 
+      {{#if (or perms.canRead perms.canDelete)}}
+        <Hds::Dropdown as |dd|>
+          <dd.ToggleButton @text={{t 'actions.manage'}} @color='secondary' />
+          {{#if (and perms.canRead @model.destination_id)}}
```

Comment:
Shouldn't we be checking update permissions?

Reply:
actually I wasn't sure, our `canUpdate` is bundled with the `canSave` in base [model](https://github.com/hashicorp/boundary-ui/blob/f65f483611105ac2735fda075a4184a77cda293d/addons/api/addon/abilities/model.js#L46) ability and we normally check for `read` before letting the user click on a resource to edit.  so I checked for canRead.  But in this case, the user wouldn't hit save so we should check for `canUpdate`?

Reply:
Right, normally we'd use `canSave` from edit view but in this case you could actually update it without read permissions straight from the list view, you could just have edit permissions for whatever reason and just clear it without even reading the alias. We should obviously test this but I would expect us to be just checking `canUpdate`